index.ts

```diff
@@ -0,0 +1,28 @@
+/* Part 1 */
+
+function isChanged(n: number): boolean{
```

Comment:
Generally good to avoid single letter variable names, more explicit is better. this is a very short method, probably fine as is, but would be good to get in the habit of having explicit methods